overview/glossary.markdown

```diff
@@ -41,74 +102,177 @@ Used to restrict when / where promises are evaluated.
+In CFEngine, there are a few notable log files:
+
+* `/var/logs/CFEngineInstall.log` - Information about the installation, especially useful if installing the package failed.
+* `/var/cfengine/outputs/` - Output logs of previous scheduled agent runs (if any).
+* `/var/cfengine/httpd/logs/error_log` - Apache errors (Mission Portal / API)
+
+#### Mission Portal (MP)
+
+The name given to the user interface used in commercial CFEngine editions,
+where all reports and progress summaries are kept.
+
 #### Namespaces
-#### Networking
+
+Namespaces allow you to define new scopes for bundles, variables, and classes.
+By using a specific name for the namespace, you can use short and generic names for the identifiers inside of it.
+
+By default, if you don't specify a namespace, you are using the namespace called `default`.
+The CMDB (group data / host specific data in Mission Portal) uses the `data` namespace, unless you specify a namespace.
+
+You can think of namespaces in a similar way as putting files inside folders, instead of having all of your files in one folder.
+The result is that things are more organized and less chances of files / classes / variables / bundles having conflicting names.
+
 #### Normal ordering
-#### Operators
-#### Pattern matching
+
+In CFEngine, the promises you write in policy files are evaluated according to a predetermined order, not from top to bottom of your policy file.
+
+#### Packages
+
+Software binaries or executable files.
+The CFEngine company compiles and tests
+software into packages suitable for different platforms.
 
 #### PCI compliance
 
 Payment Card Industry Data Security Standard (PCI DSS) is a set of requirements designed to ensure that ALL companies that process, store or transmit credit card information maintain a secure environment.
 
-#### Policy levels
+#### Platforms
+
+This usually refers to an operating system type, e.g. Linux (in its many flavours), or Windows, etc.
+Platforms are described using short identifiers, e.g. RH5, REL5, SuSE 11, SLES, etc.
+
 #### Policy server
 
 The special server that others consult for the latest policies is called the *policy server*.
 
 Typically the policy server is set by the bootstrapping process.
 
-#### Policy writing
 #### Policy
 
 A policy is a set of intentions about the system, coded as a list of promises.
 A policy is not a standard, but the result of specific organizational management decisions.
 
-#### Precedence
 #### Promise attributes
+
+As opposed to the promiser string (which is usually the unique identifier of a resource), promise attributes specify the desired specifics for that resource.
+A basic example is that if you want to ensure a file has a specific set of permissions, you would make a promise where the promiser string is the filename, and the desired permissions are specified as attributes.
+
+Sometimes referred to as promise constraints.
```

Comment:
```suggestion
Sometimes referred to as promise constraints.
As opposed to the promiser string (usually the unique identifier of a resource), promise attributes specify the desired specifics for that resource.
A basic example is that if you want to ensure a file has a specific set of permissions, you would make a promise where the promiser string is the filename and the desired permissions are specified as attributes.
```

Reply:
No - I think it's better to always put the alternate (deprecated) word at the end, it's not the most important thing.